http-conduit/Network/HTTP/Simple.hs

```diff
@@ -347,6 +348,13 @@ getRequestQueryString = H.parseQuery . H.queryString
 setRequestQueryString :: H.Query -> H.Request -> H.Request
 setRequestQueryString = H.setQueryString
 
+-- | Add to the existing query string parameters.
+--
+-- @since 2.3.5
```

Comment:
Overall looks good. Can you add a cabal file version bump and a ChangeLog entry too?